nodes/blobstream-bootstrapper.md

```diff
@@ -20,7 +20,7 @@ Before starting the bootstrapper, we will need to init the store:
 blobstream bootstrapper init
 ```
 
-By default, the store will be created in `~/.bootstrapper`. However,
+By default, the store will be created un `~/.bootstrapper`. However,
```

Comment:
might have be a typo mistake. `un`

Reply:
https://github.com/celestiaorg/orchestrator-relayer/pull/661 fixed here in source, will update in this PR now